src/GovUk.Education.ExploreEducationStatistics.Common/Model/FileInfo.cs

```diff
@@ -1,11 +1,18 @@
+using System;
+
 namespace GovUk.Education.ExploreEducationStatistics.Common.Model
 {
     public class FileInfo
     {
+        public Guid Id { get; set; }
         public string Extension { get; set; }
         public string Name { get; set; }
         public string FileName => Path?.Substring(Path.LastIndexOf('/') + 1);
         public string Path { get; set; }
         public string Size { get; set; }
+        public string? MetaFileName { get; set; }
```

Comment:
Think it would be better to have other specialised classes extending `FileInfo` rather than cramming everything here. `FileInfo` itself should really just hold the generic file details.

We do this already in the frontend with specialised types such as `DataFileInfo` which extends `FileInfo`.

Reply:
I agree, it would be good to clean this up. We already have two types of FileInfo, one common and one in the Admin.

Reply:
No I deleted the other so we only have one.

Reply:
done